src/components/organisms/Cart/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import Icon from '../../atoms/Icon';
+import cart from '../../../images/cart.svg';
+import './style.scss';
+
+const Cart = ({ updateCartState, updateMenuState, menuVisible, cartItems }) => {
+  const [cartVisible, setVisible] = useState(false);
```

Comment:
better aming would be isCartVisible or just isVisible since its a cart component

Reply:
Changed